### PR TITLE
feat(link): add link‘s typescript and docs

### PR DESCRIPTION
**Pull request by shenjunjian**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #1561 

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: packages/vue/src/link/src/pc.vue [hunk 1]
```diff
@@ -49,7 +48,7 @@ import { props, setup, defineComponent } from '@opentiny/vue-common'
 import type { ILinkApi } from '@opentiny/vue-renderless/types/link.type'
 
 export default defineComponent({
-  props: [...props, 'disabled', 'href', 'icon', 'type', 'underline', 'value', 'size'],
```
Comment (zzcr) [inline packages/vue/src/link/src/pc.vue:52]: 这里size属性为啥删除了呢

File: packages/vue/src/link/src/index.ts [hunk 2]
```diff
@@ -14,24 +14,40 @@ import template from 'virtual-template?pc'
...
 
 export default defineComponent({
   name: $prefix + 'Link',
   props: linkProps,
+  emits: {
+    click(_ev: MouseEvent) {
+      return true
+    }
+  },
+  slots: Object as SlotsType<{
```
Comment (zzcr) [inline packages/vue/src/link/src/index.ts:45]: 这里的SlotsType是从哪里引入的呢？需要验证下vue2和vue3下编译产物时候会受影响

File: packages/vue/src/link/src/pc.vue (hunk 1 above)
Comment (shenjunjian) [reply packages/vue/src/link/src/pc.vue:52]: 1、排查了一下， 在theme 包里未使用到这个size。 所以这个属性无效。
2、同样文档中也从未提size属性

 所以就删除了

File: packages/vue/src/link/src/index.ts (hunk 2 above)
Comment (shenjunjian) [reply packages/vue/src/link/src/index.ts:45]: 不引入也不影响构建结果

**Review by zzcr — APPROVED** [review]

**Pull request merged**